src/renderer/page/discover/view.jsx

```diff
@@ -240,14 +248,23 @@ class DiscoverPage extends React.PureComponent {
         <SubHeader fullWidth smallMargin />
         {!hasContent && fetchingFeaturedUris && <BusyMessage message={__('Fetching content')} />}
         {hasContent &&
-          Object.keys(featuredUris).map(
+          Object.keys(categories).map(
             category =>
-              featuredUris[category].length ? (
-                <FeaturedCategory
-                  key={category}
-                  category={category}
-                  names={featuredUris[category]}
-                />
+              categories[category].length ? (
+                category.indexOf('@') === 0 ? (
+                  <FeaturedCategory
+                    key={category}
+                    category={category.split('#')[0]}
```

Comment:
do you need to split it because you are extracting the channel name from a uri? look into using `parseURI`